src/OrdersList/OrdersList.js

```diff
@@ -164,6 +160,21 @@ function OrdersList({
     [location.search, ordersCount, toggleExportModal, visibleColumns, toggleColumn],
   );
 
+  const renderDetailsPane = useCallback((props) => (
+    <Panes
+      {...props}
+      refreshList={refreshList}
+    />
+  ), [refreshList]);
+
+  const urlParams = useMemo(() => (
+    matchPath(location.pathname, { path: `${match.path}/view/:id` })
+  ), [location.pathname, match.path]);
+
+  const isRowSelected = useCallback(({ item }) => {
+    return urlParams && (urlParams.params.id === item.id);
+  }, [urlParams]);
```

Comment:
It duplicates methods from the `OrderLinesList` component, is it possible to move it to some utils and import in the components?

Reply:
Thank you Mariia, good point! Updated code with common util - `useIsRowSelected`